src/core_ocean/shared/mpas_ocn_init_routines.F

```diff
@@ -459,6 +460,37 @@ subroutine ocn_init_routines_vert_coord(domain)!{{{
 
             vertCoordMovementWeights = 1.0_RKIND
 
+         ! Set weight tapering:
+         !  1.0 shallower than depth1
+         !  linear in between
+         !  0.0 deeper than depth2
+         elseif (config_vert_coord_movement.eq.'tapered_250m') then
+
+            depth1 = 250.0
+            depth2 = 500.0
```

Comment:
Is there a compelling reason not to make these 2 depths namelist parameters and just have `'tapered'` rather than 3 hard-coded choices?

Reply:
Thanks for looking through it. I went back and forth on that, but I think you're right. I originally thought that 'user_defined' would catch anything else, but for the ramp function your suggestion makes more sense.

Reply:
Fixed.